### DISCUSSION TITLE
Upgrade from concourse v6.7.4 to v7.4

### DISCUSSION THREAD

**Discussion by saketlj** [concourse/concourse#7782] [Help & Support]
Hi, Is there any known compatibility issue upgrading concourse v6.7.4 to v7.4 version. I am was going through all previous release notes but didn't find any relevant information yet.

If there is any one came across know bug or compatibility issue.

**Comment by taylorsilva** [comment]
The breaking changes in 7.0.0 are the only things to watch out for: https://github.com/concourse/concourse/releases/tag/v7.0.0
So have your teams check they're not using the `aggregate` step anymore. You can do a find/replace in your pipelines with `in_parallel` and everything will work the same.

**Reply by saketlj** [reply]
Thank you, let me take a look if we are using aggregate.
